ISM43362Interface.cpp

```diff
@@ -434,7 +434,7 @@ int ISM43362Interface::socket_recv(void *handle, void *data, unsigned size)
 
     if (!socket->connected) {
         _mutex.unlock();
-        return NSAPI_ERROR_CONNECTION_LOST;
+        return 0;
```

Comment:
if we end up here, I agree this is most probably an "orderly" connection closed ...